test/interaction/Issue5210.out

```diff
@@ -2,7 +2,7 @@
 (agda2-info-action "*Type-checking*" "" nil)
 (agda2-highlight-clear)
 (agda2-status-action "")
-(agda2-info-action "*All Goals*" "?0 : _9 ?1 : ?0 (n = n) Sort _9 [ at Issue5210.agda:9,20-27 ] " nil)
+(agda2-info-action "*All Goals, Errors*" "?0 : _9 ?1 : ?0 (n = n) Sort _9 [ at Issue5210.agda:9,20-27 ] ———— Errors ———————————————————————————————————————————————— Failed to solve the following constraints: Has PTS rule: (Nat , funSort Prop (_9 (n = n))) (blocked on _9) Has PTS rule: (Prop , _9 (n = _)) (blocked on _9) " nil)
```

Comment:
Seems like this change was caused by switching form `NoAbs` to `mkAbs` in `inferFunSort`.  Undoing this also undoes this change.
@arthur-adjedj

Reply:
Fixed in fcec24aeb4a782f1c2e5d0c009f1751ab968537e.